apps/wallet-dashboard/components/tiles/AssetTileLink.tsx

```diff
@@ -3,50 +3,30 @@
 
 'use client';
 
-import { ASSETS_ROUTE } from '@/lib/constants/routes.constants';
 import { AssetCategory } from '@/lib/enums';
 import { VisibilityOff } from '@iota/ui-icons';
 import { VisualAssetTile } from '.';
 import { IotaObjectData } from '@iota/iota-sdk/client';
 import { NonVisualAssetCard } from './NonVisualAssetTile';
-import { useExplorerLinkGetter } from '@/hooks';
-import Link from 'next/link';
-import { ExplorerLinkType } from '@iota/core';
 
 interface AssetTileLinkProps {
     asset: IotaObjectData;
     type: AssetCategory;
+    onClick?: (asset: IotaObjectData) => void;
```

Comment:
Doesn't make sense to make this optional as this is a Link
```suggestion
    onClick: (asset: IotaObjectData) => void;
```